core/server/data/migrations/versions/3.30/01-add-member-signin-url-permission-roles.js

```diff
@@ -0,0 +1,11 @@
+const {
+    addPermissionToRole,
+    combineTransactionalMigrations
+} = require('../../utils');
+
+module.exports = combineTransactionalMigrations(
```

Comment:
No need to combine any migrations here as you only have!

Reply:
@allouis Aah yeah 🤦 I was not sure if by default individual call created a transactional migration but just double checked and it makes sense. Updated now :)